tests/test_commands_windows.py

```diff
@@ -0,0 +1,164 @@
+import logging
+import os
+import sys
+import re
+
+import pytest
+from prefect import flow
+
+from prefect_shell.commands import shell_run_command
+
+
+@pytest.fixture
+def prefect_caplog(caplog):
+    logger = logging.getLogger("prefect")
+
+    # TODO: Determine a better pattern for this and expose for all tests
+    logger.propagate = True
+
+    try:
+        yield caplog
+    finally:
+        logger.propagate = False
+
+
+@pytest.fixture
+def prefect_task_runs_caplog(prefect_caplog):
+    logger = logging.getLogger("prefect.task_runs")
+
+    # TODO: Determine a better pattern for this and expose for all tests
+    logger.propagate = True
+
+    try:
+        yield prefect_caplog
+    finally:
+        logger.propagate = False
+
+
+# TODO: prefect futures result not being returned here, getting AttributeError: 'str' object has no attribute 'result'
+# @pytest.mark.skipif(sys.platform != "win32", reason="")
+# def test_shell_run_command_error_windows(prefect_task_runs_caplog):
+#     @flow
+#     def test_flow():
+#         return shell_run_command(command="ls this/is/invalid", return_all=True, shell="powershell")
+
+#     homedir = os.environ["USERPROFILE"]
+#     match = f"ls : Cannot find path {homedir}\\this\\is\\invalid because it does not exist."
+#     with pytest.raises(RuntimeError, match=match):
+#         test_flow().result(raise_on_failure=True)
```

Comment:
We can remove `.result()` to resolve this.

In beta, everything returned prefect futures, but haven't updated shell in some time.

Reply:
That certainly explains it! 😄